README.md

```diff
@@ -254,15 +254,15 @@ import {
     DdRum
 } from '@datadog/mobile-react-native';
 
-// Start a view with a unique view identifier, a custom view url, and an object to attach additional attributes to the view
+// Start a view with a unique view identifier, a custom view name, and an object to attach additional attributes to the view
 DdRum.startView(
-    '<view-key>',
-    '/view/url',
+    'ViewKey', // ViewKey has to be unique, for example it can be ViewName-unique-id
```

Comment:
let's be consistent in the docs: there are some lines where `<view-key>` is used still. We need to align: either `<view-key>` everywhere or `ViewKey`. Maybe the former? Add just to add an explanation.